### DISCUSSION TITLE
Question regarding the latest backend introduction

### DISCUSSION THREAD

**Discussion by 0323pin** [leftwm/leftwm#1244] [General]
Hi,

Just rebuilt from the latest HEAD and my update command failed since, now I need "at least one back-end".
That's fine, just added one and it's successful :)

Now, I see:
```
[features]
default = ["journald-log", "lefthk", "xlib", "x11rb"]
```

Is this the intended behavior? Why both back-ends by default?
I've built only with the `xlib` back-end as I would like to check if things work like before.

Thanks for any info.

**Comment by 0323pin** [comment]
Ok, for now I've reverted to 671fab081d675b25539cd6e241bc3d8063847cfb as I cannot get this to work.
Opening an issue instead.

**Comment by Eskaan** [comment]
> ```
> [features]
> default = ["journald-log", "lefthk", "xlib", "x11rb"]
> ```
> 
> Is this the intended behavior? Why both back-ends by default? I've built only with the `xlib` back-end as I would like to check if things work like before.

There is a new config option to select the backend, so we build both by default and let the user select. You can only build with one to improve compile times.

**Comment by 0323pin** [comment]
> There is a new config option to select the backend, so we build both by default and let the user select. You can only build with one to improve compile times.

Where's is this config defined?
On NetBSD I've pulled the default `config.ron` from the repo while, on Void I already had one of my own.

**Reply by Eskaan** [reply]
https://github.com/leftwm/leftwm/wiki/Config#backend
I think we might have missed to put it in the default config (un-specified entries count as default).

**Reply by 0323pin** [reply]
What I've expected 😀
But, I've found out the issue only happens when building with the `sys-log` feature enabled, see https://github.com/leftwm/leftwm/issues/1245
